modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gmos/GmosControllerSim.scala

```diff
@@ -31,7 +31,7 @@ private class GmosControllerSim[T<:SiteDependentTypes](name: String) extends Gmo
 
   override def pauseObserve: SeqAction[Unit] = sim.pauseObserve
 
-  override def resumePaused(expTime: Time): SeqAction[ObserveCommand.Result] = sim.resumePaused(expTime)
+  override def resumePaused(expTime: Time): SeqAction[ObserveCommand.Result] = sim.resumePaused
```

Comment:
Why isn't `expTime` used?

Reply:
It is passed to other instances of Instrument COntroller, because they don't know it, (they could read it back from EPICS channels). But that is not the case for InstrumentControllerSim.